android/src/main/kotlin/com/situm/flutter/ar/situm_ar/ARController.kt

```diff
@@ -12,33 +20,81 @@ class ARController(
     private val arView: SitumARPlatformView,
     private val arSceneHandler: ARSceneHandler,
     private val arMethodCallSender: ARMethodCallSender,
-) : DefaultLifecycleObserver {
+) : DefaultLifecycleObserver, ARControllerCallback {
```

Comment:
Temos unha dependencia cíclica entre ARController e ARSceneHandler, penso que por eso meteches esta interface. É boa idea, pero para min o nome sería máis ben ARSceneHandlerCallback: a interface defínese en ARSceneHandler e a acción orixínase alí; ARController implementa a interface e reacciona ante a callback.